elastic/kibana
# [Discover] Fix Data grid columns movement

## Summary

Fixes #115075

This PR fixes time filed disappiaring when it was selected and `hideTimeColumn` was set to true for data grid

### Test notes
- Set `hideTimeColumn` to true and `doc_table:legacy` to false
- Open discover
- Select some fields and then time field
- Move time field to the first place
- Time field should not be dissapiared
